--- rbgen.in	2003-10-24 03:31:21.000000000 +0200
+++ rbgen.in.oden	2007-09-09 18:38:59.000000000 +0200
@@ -6,7 +6,7 @@
 #
 # $Log: rbgen.in,v $
 # Revision 1.3  2003/10/24 01:31:21  damo
-# Patches from Eric Raymond: %prefix is implemented.  Various other small
+# Patches from Eric Raymond: %prefix is implemented. Various other small
 # changes avoid stepping on global namespaces and improve the documentation.
 #
 # Revision 1.2  2003/10/23 04:18:47  damo
